mantle/Makefile

```diff
@@ -9,8 +9,9 @@ else ifeq ($(GOARCH),aarch64)
 	GOARCH=arm64
 endif
 
-.PHONY: build test vendor
-build:
+.PHONY: build test vendor clean
+build: clean
```

Comment:
Also having `build` depend on `clean` is a *huge* regression in edit-compile-debug cycle, it forces rebuilding every time.  Why?